src/main/java/ua/mate/dto/mapper/DtoMapper.java

```diff
@@ -1,14 +1,16 @@
 package ua.mate.dto.mapper;
 
+import org.springframework.stereotype.Component;
 import ua.mate.dto.UserResponseDto;
 import ua.mate.model.User;
 
-public interface DtoMapper {
-    /**
-     * This method accepts a User and writes all his fields to a UserResponseDto.
-     *
-     * @param user - User model
-     * @return UserResponseDto
-     */
-    UserResponseDto mapByUser(User user);
+@Component
+public class DtoMapper {
```

Comment:
```suggestion
public class UserDtoMapper {
```
we will have a lot of mappers later